docs/topics/downloader-middleware.rst

```diff
@@ -1074,13 +1074,12 @@ Protego parser
 
 `Protego <https://github.com/scrapy/protego>`_ is a pure-Python robots.txt_ parser.
 The parser is fully compliant with `Google's Robots.txt Specification
-<https://developers.google.com/search/reference/robots_txt>`_ hence supports wildcard 
+<https://developers.google.com/search/reference/robots_txt>`_ hence supports wildcard
 matching, and uses the length based rule similar to `Reppy <https://github.com/seomoz/reppy/>`_.
+Scrapy uses this parser by default.
```

Comment:
It could make sense to change the order of the introduction of different parsers, so that the default one goes first.

Reply:
If we move it to the top, description of this parser shouldn't mention reppy, it would be confusing otherwise.